packages/ui/src/views/chatmessage/ChatMessage.css

```diff
@@ -118,7 +118,7 @@
 
 .cloud {
     width: 400px;
-    height: calc(100vh - 260px);
+    height: calc(90vh - 260px);
```

Comment:
90vh is little bit too small:
![image](https://github.com/FlowiseAI/Flowise/assets/26460777/b57e2761-b4ac-402e-a689-f09b99373672)

Reply:
Added fix: 
![image](https://github.com/FlowiseAI/Flowise/assets/26460777/80596ac9-8fcc-4051-a717-09326e03f1ea)